### PR TITLE
feat: pre-fill address and amount in `create-fuels` faucet

### LINKED ISSUE #2389: Add a default amount distributed from faucet
When a user interacts with the [faucet](https://github.com/FuelLabs/fuels-ts/blob/master/apps/create-fuels-counter-guide/src/hooks/useFaucet.tsx#L6) having [created a fuel example dApp](https://docs.fuel.network/docs/fuels-ts/creating-a-fuel-dapp/#creating-a-fuel-dapp) the amount should be pre-populated, this would make the UX much smoother as a user wouldn't necessarily have an idea of how much coins they should request to basic transactions.

Related: https://github.com/FuelLabs/fuels-ts/issues/2388

### PR DESCRIPTION
**Pull request by Dhaiwat10**
## Summary

After this PR, whenever someone opens the faucet page, the inputs will be pre-filled with default values so they can just click the button and get funds.

Closes #2389

### CODE REVIEW

**Review by maschad — DISMISSED** [review]

**Review by arboleya — CHANGES_REQUESTED** [review]
After the great addition below, we must consider increasing the template coverage:
 - #2278

A quick new step/assertion [here](https://github.com/FuelLabs/fuels-ts/pull/2278/files#diff-b1264f9cff227c1e32a5005039411eb40c45ec30d112a5c84f5d346333257f78R25-R26) should do it.

**Comment by Dhaiwat10** [comment]
I've filed an issue for the todo: #2494

File: playwright-tests/create-fuels.test.ts [hunk 1]
```diff
@@ -25,3 +25,28 @@ test('counter contract - increment function call works properly', async ({ page
...
+  // click the send funds button
+  const sendFundsButton = page.getByText('Send Funds');
+  await sendFundsButton.click();
+
+  await page.waitForTimeout(500);
+
+  const successToast = page.getByText('Funds sent!');
+  await expect(successToast).toBeVisible();
+
+  // TODO: Validate and assert that the balance has been updated. We need to use `fuels` as a dependency to do this.
```
Comment (Dhaiwat10) [inline playwright-tests/create-fuels.test.ts:51]: Filed an issue for this todo: #2494

Comment (arboleya) [reply playwright-tests/create-fuels.test.ts:50]: @Dhaiwat10 Could you please further elaborate on the issue description? 
 - https://github.com/FuelLabs/fuels-ts/issues/2494

It isn't obvious why we can't use `fuels` as a dependency or why exactly that would be necessary. Also, if you have an insight into a possible hypothesis/solution, that would be helpful.

Comment (Dhaiwat10) [reply playwright-tests/create-fuels.test.ts:50]: @arboleya added some context there.

Comment (Dhaiwat10) [reply playwright-tests/create-fuels.test.ts:50]: @petertonysmith94 I am not sure if that would be possible, but might be worth a try. Would need a few changes in the playwright config.

**Review by nedsalk — APPROVED** [review]

**Review by maschad — APPROVED** [review]

File: playwright-tests/create-fuels.test.ts (hunk 1 above)
Comment (Dhaiwat10) [reply playwright-tests/create-fuels.test.ts:50]: Resolving this conversation for now since I have filed an issue for this

**Review by arboleya — APPROVED** [review]

**Pull request merged**